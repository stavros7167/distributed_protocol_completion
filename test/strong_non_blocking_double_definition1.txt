strong_non_blocking [a]

environment E {
  inputs []
  outputs [a, b]
  initial e0
  e0 a! e1
  e1 b! e0
  e1 a! e0
}

process P {
  inputs [a, b]
  outputs []
  initial p0
  p0 a? p0
}

strong_non_blocking [b]
