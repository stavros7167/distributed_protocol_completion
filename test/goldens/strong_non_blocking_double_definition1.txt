Parsing file test/strong_non_blocking_double_definition1.txt
Only one strongly non blocking specification is allowed per file 19.
